Bindings/Java/Matlab/Utilities/osimVec3ToArray.m

```diff
@@ -0,0 +1,31 @@
+% ----------------------------------------------------------------------- %
+% The OpenSim API is a toolkit for musculoskeletal modeling and           %
+% simulation. See http://opensim.stanford.edu and the NOTICE file         %
+% for more information. OpenSim is developed at Stanford University       %
+% and supported by the US National Institutes of Health (U54 GM072970,    %
+% R24 HD065690) and by DARPA through the Warrior Web program.             %
+%                                                                         %
+% Copyright (c) 2005-2016 Stanford University and the Authors             %
+% Author(s): James Dunne                                                  %
+%                                                                         %
+% Licensed under the Apache License, Version 2.0 (the "License");         %
+% you may not use this file except in compliance with the License.        %
+% You may obtain a copy of the License at                                 %
+% http://www.apache.org/licenses/LICENSE-2.0.                             %
+%                                                                         %
+% Unless required by applicable law or agreed to in writing, software     %
+% distributed under the License is distributed on an "AS IS" BASIS,       %
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
+% implied. See the License for the specific language governing            %
+% permissions and limitations under the License.                          %
+% ----------------------------------------------------------------------- %
+function vec = osimVec3ToArray(p)
+% import Java Libraries
+import org.opensim.modeling.*
+% Check the input type
+if strcmp(class(p), 'org.opensim.modeling.Vec3')
+     % Convert the input Opensim Vec3 to a Matlab Vector
+     vec = [p().get(0) p().get(1) p().get(2)];
```

Comment:
Are the parentheses after `p` necessary? I'm afraid it's constructing a new Vec3. Would `p.get(0)` work?

Reply:
If p is a Vec3(), in Matlab `p(0)` leads to an exception:

```
>> p(0)
Subscript indices must either be real positive integers or logicals.
>> p().get(0)
ans = 2.3
```

Reply:
Not sure, but I think the concern was that `p()` could be performing a "do-nothing" operation on `p` and then returning a new Vec3. I think it is safest to do `p.get(0)`.

Reply:
@jimmyDunne thanks for trying that but I think you saw my `p.get(0)` as `p(0)`.

Reply:
Thanks, I did. I incorporated your suggestion.